### ISSUE TITLE
TypeError: an integer is required (got type NoneType)

### ISSUE THREAD

**Issue by halloleo** [dawsonbooth/ascii-art#2]
After pip-installing `ascii-art-cli` I ran on the command line

    ascii-art image.jpg

and got no output, just the following traceback:

```
Traceback (most recent call last):
  File "/home/broleo/opt/pipx/bin/ascii-art", line 8, in <module>
    sys.exit(main())
  File "/home/halloleo/venvs/ascii-art-cli/lib/python3.7/site-packages/ascii_art/main.py", line 36, in main
    image, args.width, args.height, args.chars, args.font, args.invert, args.normalize)
  File "/home/halloleo/venvs/ascii-art-cli/lib/python3.7/site-packages/ascii_art/__init__.py", line 43, in image_to_ascii
    image = image.resize((width, height)).convert('L')
  File "/home/halloleo/venvs/ascii-art-cli/lib/python3.7/site-packages/PIL/Image.py", line 1883, in resize
    im = im.resize(size, resample, box)
  File "/home/halloleo/venvs/ascii-art-cli/lib/python3.7/site-packages/PIL/Image.py", line 1888, in resize
    return self._new(self.im.resize(size, resample, box))
TypeError: an integer is required (got type NoneType)
```

I have used png and jpg files - with the same result.

Not sure if this helps, but the `ascii-art` did install the following packages:

* `ascii-art-cli`==1.1.0
* `Pillow`==6.2.2

**Comment by dawsonbooth** [comment]
Good catch! Sorry it took a while to get around to this. This should be fixed in [v1.2.0](https://pypi.org/pypi/ascii-art-cli/). Let me know if there are any more issues!

**Comment by halloleo** [comment]
Cool. Thanks. Works now like a beauty. :-)
